ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -40,7 +41,7 @@ export default function EditGasPopover({
   transaction,
   mode,
   onClose,
-  minimumGasLimit,
+  minimumGasLimit = `0x${decimalToHex(TWENTY_ONE_THOUSAND)}`,
```

Comment:
Instead of this, you can use `GAS_LIMITS.SIMPLE`, `GAS_LIMITS` is from the same file as `TWENTY_ONE_THOUSAND`, and then we dont need to export that or duplicate the logic

Reply:
see line 8 of shared/constants/gas.js